Druid/DRBasicBlock.class.st

```diff
@@ -318,10 +318,18 @@ DRBasicBlock >> divide: aValue by: anotherValue [
 	^ self addInstruction: (self instructionFactory divide: aValue by: anotherValue)
 ]
 
+{ #category : #dominance }
+DRBasicBlock >> dominatedBlocks [
+	"Return true if the blocks in the argument are the only dominated blocks of the receiver.
+	This checks transitive domination"
```

Comment:
This comment doesn't apply.

```suggestion
```

Reply:
True, but the transitivity still applies here. I'll change the comment :)